chia/wallet/dao_wallet/dao_wallet.py

```diff
@@ -1535,6 +1543,19 @@ async def generate_proposal_vote_spend(
                     get_innerpuzzle_from_cat_puzzle(Program.from_bytes(bytes(spend.puzzle_reveal)))
                 ).get_tree_hash()
             )
+        # vote_amounts_or_proposal_validator_hash  ; The qty of "votes" to add or subtract. ALWAYS POSITIVE.
+        # vote_info  ; vote_info is whether we are voting YES or NO. XXX rename vote_type?
```

Comment:
Please rename `vote_info` - too generic, unless that variable contained all of the vote info.